config/routes.rb

```diff
@@ -4,7 +4,7 @@
 
   resources :topics do
     resources :posts, except: [:index] do
-      resources :summaries
+      resources :summary
```

Comment:
And yeah, I decided to really double down on the whole singular resource thing. I've never done it before, but I get it, it makes sense. It does complicate things a little.